config/config_test.go

```diff
@@ -19,7 +19,6 @@ func TestSpec(t *testing.T) {
 			})
 
 			Convey("The values should be set to the expected defaults", func() {
-				So(cfg.AWSPrivateKey, ShouldEqual, "")
```

Comment:
The test should probably be updated to include the new values?

Reply:
@mattrout92 has updated this